atd-vzv/src/views/summary/CrashesByTimeOfDay.js

```diff
@@ -33,6 +33,73 @@ const hourBlockArray = [...Array(24).keys()].map((hour) =>
   moment({ hour }).format("hhA")
 );
 
+/**
```

Comment:
These two big blocks of green and red are to move some helpers outside of the component body and add some documentation. I had to add some parameters to a couple of them and then pass them where they are called.